resources/sync_gateway_configs/sync_gateway_default_functional_tests_couchbase_port_cc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "interface":":4984",
+    "adminInterface": "0.0.0.0:4985",
+    "maxIncomingConnections": 0,
+    "maxCouchbaseConnections": 16,
+    "maxFileDescriptors": 90000,
+    "slowServerCallWarningThreshold": 500,
+    "compressResponses": false,
+    {{ sslcert }}
+    {{ sslkey }}
+    {{ logging }}
+    "log": ["*"],
+    "databases":{
+        "db":{
+            {{ autoimport }}
+            {{ xattrs }}
+            {{ no_conflicts }}
+            {{ sg_use_views }}
+            {{ num_index_replicas }}
+            {{ username }}
+            {{ password }}
+            {{ cacertpath }}
+            {{ certpath }}
+            {{ keypath }}
+            "server":"couchbase://{{ couchbase_server_primary_node }}",
```

Comment:
How is this different than 'no_port', given that the port isn't being specified here?

Reply:
actually it is typo on the name. I meant couchbase protocol. Anyways I renamed and elaborated the name to understand better